rest-api/test/unit_test/api_test/participant_api_test.py

```diff
@@ -99,10 +85,35 @@ def test_update_right_ifmatch_specified(self):
     response['providerLink'] = [ self.provider_link_2 ]
     response['withdrawalStatus'] = 'NO_USE'
     response['suspensionStatus'] = 'NO_CONTACT'
+    response['site'] = 'UNSET'
+    response['organization'] = 'UNSET'
+    response['awardee'] = 'PITT'
+    response['hpoId'] = 'PITT'
     path = 'Participant/%s' % participant_id
     update_response = self.send_put(path, response, headers={ 'If-Match': 'W/"1"' })
     response['meta']['versionId'] = 'W/"2"'
     response['withdrawalTime'] = update_response['lastModified']
     response['suspensionTime'] = update_response['lastModified']
     self.assertJsonResponseMatches(response, update_response)
 
+  def test_change_pairing_awardee_and_site(self):
+    participant = self.send_post('Participant', self.participant)
+    participant['providerLink'] = [ self.provider_link_2]
+    participant_id = participant['participantId']
+    participant['awardee'] = 'PITT'
+    participant['site'] = 'hpo-site-monroeville'
+    path = 'Participant/%s' % participant_id
+    update_awardee = self.send_put(path, participant, headers={'If-Match': 'W/"1"'})
+    self.assertEquals(participant['site'], update_awardee['site'])
+    self.assertEquals(participant['awardee'], update_awardee['awardee'])
```

Comment:
It would be good to test that:

- just passing up site changes organization and awardee to ancestors
- just passing up organization changes awardee to parent, and clears out site
- just passing up awardee changes awardee and clears out site and organization